collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -611,4 +620,160 @@ public void questions_shouldHavePrefilledValue() {
                 .clickSaveAndExit();
     }
 
+    @Test
+    public void selectedAnswer_shouldBeDisplayed() {
+        //TestCase52
+        new MainMenuPage(rule)
+                .startBlankFormWithRepeatGroup("CalcTest")
+                .clickOnAddGroup(new FormEntryPage("CalcTest", rule))
+                .clickOnText("Gillnet")
+                .swipeToNextQuestion()
+                .checkIsTextDisplayed("* 7.2 What is the size of the mesh for the Gillnet ?")
+                .swipeToPreviousQuestion()
+                .clickOnText("Seinenet")
+                .swipeToNextQuestion()
+                .checkIsTextDisplayed("* 7.2 What is the size of the mesh for the Seinenet ?");
+    }
+
+    @Test
+    public void questions_shouldNotDisappear() {
+        //TestCase54
+        new MainMenuPage(rule)
+                .startBlankForm("3403_ODK Version 1.23.3 Tester")
+                .clickOnText("New Farmer Registration")
+                .scrollToAndClickText("Insemination")
+                .scrollToAndCheckIsDisplayed("New Farmer Registration");
+    }
+
+    @Test
+    public void missingFileMessage_shouldBeDisplayed() {
```

Comment:
It might be obvious for you but for someone who reads it for the first time it's not clear why so something like:
`missingFileMessage_shouldBeDisplayedIfExternalFIleIsMissing` or something similar.